memote/utils.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017 Novo Nordisk Foundation Center for Biosustainability,
+# Technical University of Denmark.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility functions used by memote and its tests."""
+
+from __future__ import absolute_import
+
+__all__ = ("register_with",)
```

Comment:
I've noticed that you add a 'hanging' comma at the end of lists a lot. What is the reason for that?

Reply:
It's a one-tuple (using `()` rather than `[]`) and it's required for tuples with only one element. Otherwise they are just parentheses around an object.